src/code-snippet/code-snippet.component.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from "@angular/core";
 
 import { I18n } from "carbon-components-angular/i18n";
+import { IconModule } from "../icon/icon.module";
```

Comment:
Same comment here, I don't think we need to import the module in the component
```suggestion
import { I18n } from "carbon-components-angular/i18n";
```

Reply:
Thanks for reviewing this. I accepted all of your suggestions.